README.rst

```diff
@@ -30,10 +30,16 @@ Policy Definition
       some/image/path/image-B-$serial:
         action: deprecate
         keep: 3
+      some/image/path/image-C-$serial:
+        action: deprecate
+        keep: 3
+        expiration_months: 3
 
 In the above example, ``some/image/path/image-A-$serial`` will find all images across all regions (owned by the current user) matching ``some/image/path/image-A-*`` where serial is replaced with a wildcard. These images will then be sorted by whatever matches in the place of $serial. The policy defined for this image is ``{action: delete, keep 1}`` meaning delete/deregister all except the latest image as defined by the sorted serials.
 
-The second image (``some/image/path/image-A-$serial``) has a policy of ``{action: deprecate, keep 3}``. Rather than deregistering the AMIs, all but the latest three will be scheduled for deprecation 1 minute in the future. These images will not be visible in the browser and will only show in API results if the caller specifies they are searching for deprecated AMIs
+The second image (``some/image/path/image-B-$serial``) has a policy of ``{action: deprecate, keep 3}``. Rather than deregistering the AMIs, all but the latest three will be scheduled for deprecation 1 minute in the future. These images will not be visible in the browser and will only show in API results if the caller specifies they are searching for deprecated AMIs
+
+The third image (``some/image/path/image-C-$serial``) has a policy of ``{action: deprecate, keep 3, expiration_months 3}``. All but the three most recent images will be scheduled for deprecation (like in the previous example). In addition, any image older than 3 months, besides the newest one, will also be scheduled for deprecation. 
```

Comment:
my interpretation of the requirement is that we will keep the following:
* all images less than 3 months old will be retained (i.e. not cleanup action taken).
* keep a minimum of 3 
* mark everything else as deprecated (or deleted, based on action)

the docs don't seem to match the intent